docs/src/pages/guides/interoperability.md

```diff
@@ -231,47 +234,50 @@ function StyledComponentsButton() {
 
-In some cases, the approaches above will not work. For example, if one attempts to style a [Drawer](/demos/drawers/) with variant `permanent`, one will likely need to affect the Drawer's underlying `paper` style. However, this is not the root style of `Drawer` and therefore styled-components customization as above will not work. One can workaround this by using [stable JSS class names](/customization/css-in-js/#class-names), but less dangerous approach is to use the `classes` property to introduce an override style, and then style it with higher specificity via `&`.
+In some cases, the approaches above will not work.
+For example, if you attempt to style a [Drawer](/demos/drawers) with variant `permanent`,
+you will likely need to affect the Drawer's underlying `paper` style.
+
+However, this is not the root element of `Drawer` and therefore styled-components customization as above will not work.
+You can workaround this by using [stable JSS class names](/customization/css-in-js/#global-css), but the most reliable approach is to use the `classes` property to introduce an override style, and then style it with higher specificity via `&`.
 
 The following example overrides the `label` style of `Button` in addition to the custom styles on the button itself. It also works around [this styled-components issue](https://github.com/styled-components/styled-components/issues/439) by "consuming" properties that should not be passed on to the underlying component.
 
 ```jsx
-import React from "react";
-import styled from "styled-components";
-import Button from "material-ui/Button";
+import React from 'react';
+import styled from 'styled-components';
+import Button from 'material-ui/Button';
 
-const StyledButton = styled(({ cornerRadius, ...rest }) => (
-  <Button {...rest} classes={{ label: "label-override" }} />
+const StyledButton = styled(({ color, ...other }) => (
+  <Button {...other} classes={{ label: 'label' }} />
 ))`
   background: linear-gradient(45deg, #fe6b8b 30%, #ff8e53 90%);
-  border-radius: 3px;
+  border: 0;
   color: white;
+  height: 48px;
   padding: 0 30px;
+  box-shadow: 0 3px 5px 2px rgba(255, 105, 135, 0.3);
 
-  & .label-override {
-    border: 5px dotted black;
-    border-radius: ${props => props.cornerRadius}px;
-    padding: 0 30px;
+  & .label {
+    color: ${props => props.color};
   }
 `;
 
 function StyledComponentsButton() {
   return (
     <div>
       <Button>Material-UI</Button>
-      <StyledButton cornerRadius={10}>Styled Components</StyledButton>
+      <StyledButton color="papayawhip">Styled Components</StyledButton>
     </div>
   );
 }
 
 export default StyledComponentsButton;
 ```
 
-TODO edit button here
-
-Lastly, wrap the styled-component with `withStyles` in order to use the material-ui `theme` prop.
+[![Edit Button](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/j4n13yl1r9)
```

Comment:
@oliviertassinari The code at the link here does not match the code sample above.

Reply:
I think it does. What make you think so?

Reply:
@oliviertassinari This is what I get at that URL, and you can see the code snippet does not match:

![image](https://user-images.githubusercontent.com/53049/36222772-62ba1718-1190-11e8-8841-dc86048f6ed1.png)

Reply:
We miss the `initialPath` for all the demos 
https://codesandbox.io/embed/j4n13yl1r9?initialpath=%2FStyledComponentsButton.js

Reply:
But the nice aspect is that it's showing people how the override issue can be handled.

Reply:
Indeed, my error. Thanks!

Reply:
There a much easier way to do this: https://codesandbox.io/s/9o3n53ppqp

```
const StyledButton = styled(Button)`
  background: linear-gradient(45deg, #fe6b8b 30%, #ff8e53 90%);
  border: 0;
  color: white;
  height: 48px;
  padding: 0 30px;
  box-shadow: 0 3px 5px 2px rgba(255, 105, 135, 0.3);

  > .label {
    color: ${props => props.color};
  }
`;
```

Reply:
@designjudo I agree 👍. It's simpler in this case, but it doesn't scale. This demo is here to show a pattern, not to solve this specific case.